### ISSUE TITLE
Justinian Bookmark(s)

### ISSUE THREAD

**Issue by loup99** [rickinator9/WTWSMS#14]
Thi is the issue dedicated to Justinian and the bookmarks regarding him. History, province ownership, a bookmark, everything goes into here. It is partially implemented, but could still use some tweaking and balancing aswell as play-testing.

**Comment by EWattman** [comment]
The Ostrogoths hold half of Dacia for some reason
The Visigoths haven't let go of Melguiel yet
Aneb in Tripolitania is blank
The Governor of Dalmatia holds the title of Exarch of Ravenna?
Shouldn't the African Prefecture be a Kingdom?
The Gepids are under a Dacian King and they have seized Birlad and Galaz from the Sklaveni

**Comment by loup99** [comment]
So aneb lacks a holder, but what is the issue with Melgueil?

**Comment by EWattman** [comment]
The Visigoths aren't supposed to hold Melgueil in 535. And might I add that there is no war between the Empire and the Goths

**Comment by loup99** [comment]
The War is not weird, the bookmark actually starts before the actual war. So either we rename it to "Justinian", or we move it.

**Comment by EWattman** [comment]
And what about the African Prefecture?

**Comment by loup99** [comment]
No idea about the prefecture, but is it like the kingdoms we have at start?

 The exarchate? Kingdom most likely, or atleast how I see it worked.

**Comment by IhateTrains** [comment]
"The Gepids are under a Dacian King and they have seized Birlad and Galaz from the Sklaveni"

Is it a bug report or historical suggestion?

**Comment by loup99** [comment]
All of those were issues, and I believe all have been fixed.

**Comment by IhateTrains** [comment]
I'm not sure if "The Ostrogoths hold half of Dacia for some reason" is fixed

**Comment by IhateTrains** [comment]
Can this issue be closed then?

**Comment by loup99** [comment]
This issue wasn't dedicated to those particular issues but rather the bookmark in general. I think it is still relevant, there is a lot which could be improved.

**Comment by IhateTrains** [comment]
Do you have any particilar ideas in mind? I could improve something for 0.9.12 (the release after the next release)

**Comment by IhateTrains** [comment]
![2017-12-20 4](https://user-images.githubusercontent.com/29546927/34221268-b76f53cc-e5b7-11e7-9a58-4664951edc1c.png)
Why do the Ostrogoths hold the area in purple? Do you have any source for this?

**Comment by loup99** [comment]
A few thoughts:

- Chariot racing, which should be getting added by @CaudilloFranco, had a major impact on the reign of Justinian with most notably the Nika riots. The main teams of the Greens and Blues in the capital had a lot of influence. 
- Correct me if I'm wrong, but some characters of the reign such as [Tribonian](https://en.wikipedia.org/wiki/Tribonian) and [Mundo ](https://en.wikipedia.org/wiki/Mundus_(general)) seem to be missing.
- The biggest issue though, in my view, is the fact that most characters have a far too big demesne (this actually doesn't apply to Justinian himself). See the prefect of Africa, Theodahad or the respective Frankish kings as examples of way too big demesnes.  I know this might seem like a bad excuse to keep this issue in particular open when it applies to others too, but I still think it is a problem in 476 as well.

**Comment by IhateTrains** [comment]
I can create Tribonian and Mundo character.

**Comment by loup99** [comment]
Note that Mundo might be a bit more difficult to add than Tribonian, since part of his family is already in.

**Comment by IhateTrains** [comment]
I have found his niece, will check if his father is in.

**Comment by loup99** [comment]
If you have access to it, they should all be in the PLRE.

**Comment by IhateTrains** [comment]
For the huge demesnes, I suggest finding historical vassals or creating fictional vassals based on administrative divisions.

**Comment by IhateTrains** [comment]
I will post historical characters that can be added to improve the VI century (will be updated every time I find a new character:
- [ ] PLRE vol. 3 Sigivaldus 1, dux of the Auvergne c. 525-533

**Comment by loup99** [comment]
Agreed on the huge demesnes and finding characters is certainly needed, so good work there.

**Comment by IhateTrains** [comment]
Do you have any ideas on finding vassals for Ostrogothica?

**Comment by Abian36** [comment]
IDK, I write in the PLRE list every character I find, so if there are any they must be there.

**Comment by loup99** [comment]
We can add military leaders as vassals in areas of Ostrogothic implementation.

**Comment by Abian36** [comment]
Wisibadus (Page 1407 VOL III) A Goth, Comes of Ticinum/Pavia 535/536 and after

**Comment by Abian36** [comment]
Also found the historical duke of Pavia during rule of dukes:
Zaban (Page 1409 VOL III) Lombard dux of Pavia during rule of Dukes

**Comment by loup99** [comment]
Wrong thread, and we already have that duke.

**Comment by Abian36** [comment]
Oh, I'm sorry.

**Comment by IhateTrains** [comment]
![image](https://user-images.githubusercontent.com/29546927/50529963-59214d80-0af9-11e9-9b45-6f280beaaf3e.png)
Seconding @EWattman, why do the Ostrogoths hold that area east of the Danube? I can fix it if the team agrees,

**Comment by tannerflick4** [comment]
I don't remember them ever having much sway over the east bank of the Danube, perhaps give that land to the Gepids? most of what I know show Gepids being more "rounded" in shape of borders than what we have for this date.

**Comment by IhateTrains** [comment]
Yes, I would probably give it to the Gepids.

**Comment by IhateTrains** [comment]
Another thing I can work on is preventing the Gepids from taking the two provinces from the Sklaveni and splitting them in half.

**Comment by IhateTrains** [comment]
The issue with exarchate of Ravenna has been fixed for this particular start date.

**Comment by loup99** [comment]
@IhateTrains @tannerflick4 I believe all tribes are playable as of the latest release. Adding a task list to the opening post for remaining progress on this.

**Comment by IhateTrains** [comment]
I've found some remaining unplayable countries due to dead rulers.
![image](https://user-images.githubusercontent.com/29546927/79168503-60ea2b80-7dea-11ea-80f4-a9b5fb6c9123.png)
![image](https://user-images.githubusercontent.com/29546927/79168522-6cd5ed80-7dea-11ea-9d3d-586be5cb64a8.png)
![image](https://user-images.githubusercontent.com/29546927/79168571-8c6d1600-7dea-11ea-8e92-f223c01554d8.png)
![image](https://user-images.githubusercontent.com/29546927/79168611-a3ac0380-7dea-11ea-8fa2-153a19fbc78d.png)
![image](https://user-images.githubusercontent.com/29546927/79168702-d655fc00-7dea-11ea-86ea-a0e5b3e5526d.png)
![image](https://user-images.githubusercontent.com/29546927/79168749-f5548e00-7dea-11ea-85fd-65ccc2f4e5aa.png)
![image](https://user-images.githubusercontent.com/29546927/79168765-000f2300-7deb-11ea-91f5-f3257f41a31d.png)
![image](https://user-images.githubusercontent.com/29546927/79168877-3c428380-7deb-11ea-9851-fced31043258.png)
![image](https://user-images.githubusercontent.com/29546927/79168931-52504400-7deb-11ea-9db6-8c1477d691c0.png)

All this
![image](https://user-images.githubusercontent.com/29546927/79168661-be7e7800-7dea-11ea-9814-af84e478f47c.png)

**Comment by loup99** [comment]
Thank you for signalling that. The easiest to fix will be West Africa since I know for sure that all rulers there are fictional.

**Comment by Symmachus** [comment]
By the way Spanish duchies and Persian satraps are barely playable at that date since their rulers are 65 YO and don't have any children.

**Comment by Abian36** [comment]
I have been researching Irish nobility for the past days so if you give me a bit of time I can attempt to fill those holes with Historical Characters

**Comment by loup99** [comment]
> By the way Spanish duchies and Persian satraps are barely playable at that date since their rulers are 65 YO and don't have any children.

For characters who historically don't have families you can use the vanilla game rule to generate some.

> I have been researching Irish nobility for the past days so if you give me a bit of time I can attempt to fill those holes with Historical Characters

Ok, but what sources have you been using? Are you aware of sites like https://www.historyfiles.co.uk/ ?

**Comment by Abian36** [comment]
That source doesn't mention anything Ireland related but it can be useful for later Welsh Celtic History

**Comment by Abian36** [comment]
I am using this one: http://sites.rootsweb.com/~irlkik/ihm/leinst4.htm

**Comment by loup99** [comment]
> That source doesn't mention anything Ireland related but it can be useful for later Welsh Celtic History

It does have some stuff on Ireland if you click on the different Irish realms of this map: https://www.historyfiles.co.uk/FeaturesBritain/BritishMap.htm

**Comment by Symmachus** [comment]
Another issue at this bookmark is d_senate. The only consul for 535 was Belisarius but if we give him d_senate, he will became Ostrogothic vassal. At the moment when you start new game, Senate will be destroyed and replaced by the Urban prefecture and patricians transform into standard mayors and barons. I would suggest to add non-historical consul for that date, choosing from one of the patricians.
